pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "mm_image_processing"
+version = "0.1.0"
+packages = [] # add required pacakges later?
```

Comment:
This should probably explicitly list `mm_image_processing` as a module provided by this package

Reply:
Included this.